tests/test_images.py

```diff
@@ -26,52 +31,60 @@
     remote = ("https://github.com/aio-libs/aiodocker/"
               "raw/master/tests/docker/docker_context.tar")
 
-    tag = "image_from_tar:1.0"
+    tag = "{}:1.0".format(_random_name())
     params = {'tag': tag, 'remote': remote, 'stream': True}
     stream = await docker.images.build(**params)
 
-    async for output in stream:
-        if "Successfully tagged image_from_tar:1.0\n" in output:
-            break
+    async for _ in stream:  # noqa: F841
```

Comment:
We can remove this loop, is not need if we don't check the stream.
The only thing you have to change is `stream:False`